### PR TITLE
add landing page

### PR DESCRIPTION
**Pull request by JohnL17**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.pyc
```
Comment (anehx) [inline .gitignore]: Why would you ignore python cache files if there are absolutely no python files?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+![Landing Page Screenshot](/screenshot.png)
+
+# Freedomvote Landing Page
```
Comment (anehx) [inline README.md]: Swap title and screenshot

File: index.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Freedomvote</title>
+        <link rel="stylesheet" href="style.css">
```
Comment (anehx) [inline index.html]: More meta tags, like description etc..

**Review by anehx — CHANGES_REQUESTED** [review]
Some nitpicks, otherwise looks good. Does it scale to mobile?

File: .gitignore (hunk 1 above)
Comment (JohnL17) [reply .gitignore]: Ah right. I didn't consider that.

**Review by anehx — APPROVED** [review]

**Pull request merged**